everyone-on-support.md

```diff
@@ -12,6 +12,8 @@ In the end, we are working for our customers, and having everyone involved in th
 
 * We use [GrooveHQ](https://niteoweb.groovehq.com/) for handling support emails. For EOS, we have predefined account which you'll need to personalize when you're on duty. Login credentials are on 1Password under *Team* vault. Go to *Edit Profile page* and update the *Name* field, to your personal name(Note: Your name will not be reflected in UI after you change it, until you logout and login back again, so do that first) and leave the username as is (@Niteo).
 * After logging back to GrooveHQ, you will see the list of all tickets with assigned agent next to it, choose one where no one is assigned to it, and assign it to yourself.
+* When replaying select *Send as Open*, so that ticket is properly tracked.
```

Comment:
`replying` probably?

Reply:
How is it properly tracked with "Send as open"? That keeps it open, instead of closed.

Reply:
You don't see it assigned to you? I know that docs state that you should close the ticket as soon as possible, but in cases where you need a response, you still need to keep it open and a default is closed.

Reply:
Send as Pending then, you can't keep it open because there will be too many tickets under open.

Proposals for changes in the process should be done in a ticket and discussed, not immediately changed in the handbook.

Reply:
@dz0ny You're complicating things with those above. What only needs to be added is to add instruction to ping marbeann when you're reporting for EOS and give her at least two minutes to talk.